src/shared/components/login/login.js

```diff
@@ -157,7 +164,7 @@ class Login extends Component {
       <Section title="Login" theme="white">
         <Form autoComplete>
           <FormEmail id="email" displayName="Email" label="Email" onChange={this.onEmailChange} />
```

Comment:
Could you add the handler here as well so enter works on any field?

Do you think it makes sense to push this up to the formInput so any it works on any field we use?